plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,20 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewGauge(
+		prometheus.GaugeOpts{
```

Comment:
> I think we're not doing anywhere else, as having this graph requires running prometheus server, or something else that scrapes timeseries.

I thought even for histograms, we need to do it this way - like having a prometheus server or something, isn't it the case?

> As all our SLIs speak about percentiles we generally use Histogram for any performance-related metrics.

I am fine with making it a histogram.

ping @jayunit100 - as we initially thought of using gauge.